plugins/gatsby-source-wagtail/test/mock.json

```diff
@@ -43,4 +80,4 @@
     "idp_slug": "example-site-idp",
     "type": "pages.ProgramPage"
   }
-]
+]
```

Comment:
nit: should have a new line at the end

Reply:
Stupid prettier. Nice catch -- Even the linter didn't seem to care about that one.